crisischeckin/Services/SMSMessageSender.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using Services.Interfaces;
+using Twilio;
+
+namespace Services
+{
+    public class SMSMessageSender : IMessageSender
+    {
+        private readonly string _senderPhone;
+        private readonly Func<TwilioRestClient> _twilioClient;
+
+        public SMSMessageSender(Func<TwilioRestClient> twilioClient, string senderPhone)
+        {
+            _twilioClient = twilioClient;
+            _senderPhone = senderPhone;
+        }
+
+        public void SendMessage(Message message, IReadOnlyCollection<MessageRecipient> recipients, string senderDisplayName = null)
+        {
+            TwilioRestClient twilioClient = _twilioClient();
```

Comment:
In this code, if we are in the development environment, instead of sending text, write a text file with the information that will help people make sure their code works (the text recipients, and the content of the message)